src/renderer/windows/main/components/Snippets.vue

```diff
@@ -3,8 +3,8 @@
         <div class="flex flex-col flex-2 p-8">
             <h1 class="flex items-center h-12">
                 <span class="flex-1">
-                    <span class="text-3xl">Snippets</span>
-                    <router-link :to="{ name: 'Preferences' }" class="text-2xl text-grey-dark cursor-pointer no-underline ml-4">Preferences</router-link>
+                    <span tabindex="1" class="text-3xl">Snippets</span>
```

Comment:
This is a non-focusable target. It's just a span with no meaning in this context. It shouldn't have a `tabindex` attribute.